resources/definitions/fdmprinter.def.json

```diff
@@ -2929,6 +2929,17 @@
                     "settable_per_extruder": true,
                     "settable_per_meshgroup": true
                 },
+                "force_first":
+                {
+                    "label": "Start Layer with this Mesh",
+                    "description": "Start each layer by printing this model rather than other models. In multi-extrusion this model will be printed first when we switch to the extruder with which this model is printed. This setting doesn't affect when we switch to a different extruder.",
```

Comment:
You should add a note here how this behaves with respect to the start X/Y position of a layer, namely that this overrides the start X/Y (when applicable). Also in the start X/Y we should put a message that the setting may get overwritten by this setting.